src/ralph/networks/admin.py

```diff
@@ -111,7 +111,7 @@ def get_queryset(self, request):
 class NetworkAdmin(RalphMPTTAdmin):
     ordering = ['min_ip', '-max_ip']
     change_form_template = 'admin/data_center/network/change_form.html'
-    search_fields = ['name', 'address', 'remarks']
+    search_fields = ['name', 'address', 'remarks', 'vlan']
```

Comment:
Could you please also make it possible to filter by VLAN range by extending ```list_filter``` property with ```'vlan'```? You may put it into any position that you think will be the best.

Reply:
OK, should be in 37415d16bc04fbe41bcf4fe754131b601eb234b0

Reply:
Thanks!